pom.xml

```diff
@@ -52,22 +52,19 @@
 	</developers>
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<java.version>1.8</java.version>
-		<maven.version>3.6.2</maven.version>
-		<maven-resolver.version>1.4.1</maven-resolver.version>
-		<maven-wagon.version>3.5.2</maven-wagon.version>
-		<commons-compress.version>1.26.1</commons-compress.version>
-		<commons-io.version>2.15.1</commons-io.version>
-		<kubernetes-fabric8-client.version>5.12.4</kubernetes-fabric8-client.version>
-		<spring-boot.version>2.7.18</spring-boot.version>
-		<spring-framework.version>5.3.39</spring-framework.version>
+		<java.version>17</java.version>
+		<!-- @TODO boot3 waiting for 4.x to support jakarta ee -->
+		<maven.version>3.9.6</maven.version>
+		<!-- @TODO boot3 waiting for 4.x to support jakarta ee -->
+		<maven-resolver.version>1.9.18</maven-resolver.version>
+		<maven-wagon.version>3.5.3</maven-wagon.version>
+		<commons-io.version>2.16.1</commons-io.version>
+		<kubernetes-fabric8-client.version>6.9.2</kubernetes-fabric8-client.version>
```

Comment:
Is this fabric8 client version something we already tested against?

Reply:
I did run main-3 ITs when I did the initial work. However now docker rate limiting is so back that we need to update the ITs to provide imagePullSecrets everywhere.